scalableminds/webknossos
# No four bit segmentation

This PR ensures that segmentation data is never requested in four bit mode. Also, it changes the saving code for segmentation buckets so that the fourBit flag is not send along with the request, since that property is ignored by the server anyway.
This hopefully fixes #2811 as the only symptom we could reproduce on that is fixed with this PR.

I also merged the bucketbuilder file into the wkstore adapter since that file is only used for the wkstore anyway.

### Mailable description of changes:
 - A bug which caused segmentation data to be requested as four-bit when four-bit-mode was enabled is fixed now.

### URL of deployed dev instance (used for testing):
- https://nofourbitsegmentation.webknossos.xyz

### Steps to test:
- test a normal dataset with and without four bit mode (ensure that data is loaded with the correct flag)
- load a dataset with segmentation and ensure that segmentation is properly shown even in four-bit mode
- repeat the last step while creating a new segmentation layer

### Issues:
- contributes (?) to #2811 

------
